packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/__tests__/quanticSearchBoxInput.test.js

```diff
@@ -0,0 +1,344 @@
+
+      const submitButton = element.shadowRoot.querySelector(
+        selectors.searchBoxSubmitBtn
+      );
+      const clearIcon = element.shadowRoot.querySelector(
+        selectors.searchBoxClearIcon
+      );
+      const textarea = element.shadowRoot.querySelector(
+        selectors.searchBoxTextArea
+      );
+
+      expect(textarea).not.toBeNull();
+      expect(textarea.placeholder).toEqual(defaultOptions.placeholder);
+      expect(submitButton).not.toBeNull();
+      expect(clearIcon).toBeNull();
+    });
+  });
+
+  describe('when the withoutSubmitButton is set to false', () => {
+    it('should not display the searchIcon to the left of the searchbox', async () => {
+      const element = createTestComponent({
+        ...defaultOptions,
+        withoutSubmitButton: false,
+      });
+      await flushPromises();
+
+      const searchIcon = element.shadowRoot.querySelector(
+        selectors.searchBoxSearchIcon
+      );
+
+      expect(searchIcon).toBeNull();
+    });
+
+    it('should display the submit button to the right of the searchbox', async () => {
+      const element = createTestComponent({
+        ...defaultOptions,
+        withoutSubmitButton: false,
+      });
+      await flushPromises();
+
+      const submitButton = element.shadowRoot.querySelector(
+        selectors.searchBoxSubmitBtn
+      );
+
+      expect(submitButton).not.toBeNull();
+    });
+  });
+
+  describe('when the withoutSubmitButton is set to true', () => {
+    it('should display the searchIcon to the left of the searchbox', async () => {
+      const element = createTestComponent({
+        ...defaultOptions,
+        withoutSubmitButton: true,
+      });
+      await flushPromises();
+
+      const searchIcon = element.shadowRoot.querySelector(
+        selectors.searchBoxSearchIcon
+      );
+
+      expect(searchIcon).not.toBeNull();
+      expect(searchIcon.classList.contains('slds-input__icon_left')).toBe(true);
+    });
+
+    it('should not display the submit button to the right of the searchbox', async () => {
+      const element = createTestComponent({
+        ...defaultOptions,
+        withoutSubmitButton: true,
+      });
+      await flushPromises();
+
+      const submitButton = element.shadowRoot.querySelector(
+        selectors.searchBoxSubmitBtn
+      );
+
+      expect(submitButton).toBeNull();
+    });
+  });
+
+  describe('when the placeholder property receives a custom placeholder value', () => {
+    it('should display the custom value as the searchbox placeholder', async () => {
+      const customPlaceholder = 'Custom placeholder';
+      const element = createTestComponent({
+        ...defaultOptions,
+        placeholder: customPlaceholder,
+      });
+      await flushPromises();
+
+      const input = element.shadowRoot.querySelector(selectors.searchBoxInput);
+
+      expect(input.placeholder).not.toEqual(defaultOptions.placeholder);
+      expect(input.placeholder).toEqual(customPlaceholder);
```

Comment:
```suggestion
      expect(input.placeholder).toEqual(customPlaceholder);
```
By definition if it's equal to `customPlaceholder` it's not equal to the default placeholder ;)

Reply:
x)